doc/api/timers.md

```diff
@@ -363,6 +363,26 @@ added: v15.0.0
   * `signal` {AbortSignal} An optional `AbortSignal` that can be used to
     cancel the scheduled `Immediate`.
 
+### `timersPromises.setInterval([delay[, value[, options]]])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `delay` {number} The number of milliseconds to wait between iterations.
+   **Default**: `1`.
+* `value` {any} A value with which the iterator returns.
+* `options` {Object}
+  * `ref` {boolean} Set to `false` to indicate that the scheduled `Timeout`
+    between iterations should not require the Node.js event loop to
+    remain active.
+    **Default**: `true`.
+  * `signal` {AbortSignal} An optional `AbortSignal` that can be used to
+    cancel the scheduled `Timeout` between operations.
+  * `throwOnAbort` {boolean} Set to `true` to indicate that the iterator
```

Comment:
I don't think this should be an option - I think it should always `throw` on abort to align this behaviour with abort signal on streams.

What do you think @ronag @mcollina @jasnell ?

Reply:
I agree.

Reply:
removed `throwOnAbort`